eva-server/src/main/java/uk/ac/ebi/eva/server/security/authorization/UnsecureConfiguration.java

```diff
@@ -0,0 +1,23 @@
+package uk.ac.ebi.eva.server.security.authorization;
+
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
+import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
+
+@ConditionalOnProperty(prefix = "eva.server.security", name = "enabled", matchIfMissing = true, havingValue = "false")
```

Comment:
If I understand [this documentation](http://docs.spring.io/spring/docs/current/spring-framework-reference/html/beans.html#beans-definition-profiles-java) correctly, they also allow to configure which `@Configuration` beans to inject.